backend/data_tools/data/user_data.json5

```diff
@@ -137,8 +135,8 @@
         "POST_RESEARCH_PROJECT",
 
         "GET_USER",
-        "GET_USERS",
-
+        "PUT_USER",
+        "PATCH_USER",
```

Comment:
@johndeange Up to this point I had presumed that `USER` only affected the the same user as the one doing the verb whereas `USERS` would be something affecting a user other than themselves. I'm guessing that's not correct or at least not any longer. ?

Reply:
The Permissions are currently all singular.  I think eventually `GET_USER` will allow access to the `GET` `/users/<id>` endpoint and `GET_USERS` will allow access to the `GET` `/users` endpoint.